packages/scss/src/components/card/component.scss

```diff
@@ -3,9 +3,11 @@
 	border-radius: var(--commons-borderRadius-L);
 	border-color: var(--commons-divider-color);
 	border-width: 1px;
+	display: block;
```

Comment:
`display: flex` ?

Reply:
It would have impact on cards with `card-content` and `card-footer`.  It would be doable by adding a `flex-direction`, but I a bit afraid of side effects.